spec/rspec/core/formatters/base_text_formatter_spec.rb

```diff
@@ -35,6 +35,11 @@
       expect(formatter_output.string).to match("1 example, 1 failure, 1 pending")
     end
 
+    it "with 1s outputs singular (only pending)" do
```

Comment:
this one is unrelated right? (Don't mind including it, just making sure I understand)